app/src/main/java/nextstep/payments/ui/PaymentCard.kt

```diff
@@ -50,74 +55,59 @@ fun PaymentCard(
 @Composable
 fun PaymentCard(
     card: Card,
-    cardColor: Color,
     modifier: Modifier = Modifier,
+    content: @Composable () -> Unit = {}
 ) {
     Box(
-        contentAlignment = Alignment.BottomEnd,
-        modifier = modifier
-            .shadow(8.dp)
-            .size(width = 208.dp, height = 124.dp)
-            .background(
-                color = cardColor,
-                shape = RoundedCornerShape(5.dp),
-            )
-            .padding(bottom = 10.dp)
+        modifier = Modifier.padding(top = 10.dp)
```

Comment:
컴포넌트 파라미터로 전달받은 Modifier를 사용해야 할 것 같아요 ㅎㅎ
```suggestion
        modifier = modifier.padding(top = 10.dp)
```